#168843786 Add responsiveness to the UI

iregide opened the pull request:
#### What does this PR do?
1. Implement the following feedbacks
     - fix every content in the center of the page{-nav-links-body content
     - footer to remain at the bottom
     - fixing navigation links on some pages
     - reducing some contents in the signup page
  
#### Description of Task to be completed?
     - fixing the navigation links
     - centering home page contents
     - makes the footer to the bottom
     - fixing the single article page
     - fixing the view articles page
    
    
#### How should this be manually tested?
    1. clone the project
    2. Unzip it
    3. open the zipped file and run index.html
    4. Navigate in all the pages 

#### Any background context you want to provide?
      N/A

#### What are the relevant pivotal tracker stories?
   [PT link] (https://www.pivotaltracker.com/n/projects/2395939/stories/168843786)
#### Screenshots (if appropriate)

![ui feedbacks](https://user-images.githubusercontent.com/53136152/65924498-4bcd2280-e3ed-11e9-887b-51ce00c9207c.jpg)
![ui feedbacks2](https://user-images.githubusercontent.com/53136152/65924499-4c65b900-e3ed-11e9-9214-a4bee733ff89.jpg)
![edditeed](https://user-images.githubusercontent.com/53136152/65967180-81135800-e461-11e9-95f9-69a60d288b3d.jpg)
![editeed2](https://user-images.githubusercontent.com/53136152/65967181-81135800-e461-11e9-8aa8-e067490d488a.jpg)
![editeed](https://user-images.githubusercontent.com/53136152/65967183-81135800-e461-11e9-8f84-a1a3d5a291cc.jpg)
### Updates
![1 icon](https://user-images.githubusercontent.com/53136152/66043039-62bf6200-e51e-11e9-9bc9-bda43cf2df87.jpg)
![1](https://user-images.githubusercontent.com/53136152/66043040-6357f880-e51e-11e9-96fd-eeaf4ef34fb8.jpg)
![2 ICON](https://user-images.githubusercontent.com/53136152/66043041-6357f880-e51e-11e9-97fc-97f9c565ab4c.jpg)

#### Questions:
     N/A

iregide committed:
bug(feedbacks):add bg-ui-feedbacks-168843786

-fixing the navigation links
-centaling home page contents
-makes the footer to the bottom
-fixing the singlearticle page
-fixing the viewarticles page

[finishes#168843786]

Oluwasegun-AA commented:
@IRSAGE , you are using too many images, which doesn't make the UI look impressive. kindly remove the background image and make the UI more simple. please let me know when you are done

iregide committed:
bug(feedbacks):add bg-ui-feedbacks-168843786
-Updating the LFA'S feedbacks
-Changing colors
-remove the background image
[finishes#168843786]

iregide committed:
bug(feedbacks):add bg-ui-feedbacks-168843786

-Updating the LFA'S feedbacks
-Makes the latest article divin the center
adds icon to make it look more professional and presentable
[finishes#168843786]
